### DISCUSSION TITLE
Adguard a przyzwoitość w czasie wojny Ukraina - Rosja

### DISCUSSION THREAD

**Discussion by seba2282** [MajkiIT/polish-ads-filter#20747] [Q&A]
Czy w dobie wojny Ukraina - Rosja jest przyzwoicie używać Adguard, Kaspersky, itp. co rosyjskie?

**Comment by AroKol78** [comment]
Moim zdaniem nie jest przyzwoicie ani moralne używanie żadnych produktów pochodzących z Rosji w tym firm rosyjskich które przeniosły siedziby do innych państw lub zmienili szyld albo właściciela tylko na papierze ~papieże~.
Tym bardziej że sami Rosjanie nie zrobili wiele by nie było wojny
(proszę nie usprawiedliwiać Rosjan brakiem rzetelnej informacji, my jakoś se poradziliśmy z komuną).

**Reply by seba2282** [reply]
Popieram. W pełni twoją wypowiedź. 🙂

**Reply by hawkeye116477** [reply]
To jak tam odinstalowaliście już WinRara i 7-Zipa?

Tylko czym tu zastąpić, może programem PeaZip (twórca jest Włochem, ale to jest fork 7-Zipa) albo HaoZip (chiński)? Jeśli żaden z nich wam nie odpowiada, to lepszego wyboru chyba brak. No chyba, że jednak zgadzacie się z autorem artykułu `https://www.dobreprogramy.pl/@antar/rosyjskie-programy-i-dystrybucje-z-ktorych-nie-wypada-juz-korzystac,blog,115705` i nie wymieniacie ich.

**Reply by seba2282** [reply]
Przymierzam się do wywalenia 7zip. Ale czekaj, krąży info, że Chiny wspierają Rosjan. Ja też się zastanawiam czym zastąpić mój Huawei. 😂😂😂😂

**Reply by AroKol78** [reply]
trzeba od czegoś zacząć, droga długa i kręta (ropa,gaz,...), łatwo nie będzie.

**Reply by hawkeye116477** [reply]
Odnośnie https://github.com/MajkiIT/polish-ads-filter/discussions/20747#discussioncomment-2338342, to i jedno i drugie pewnie nie byłoby takie proste, bo większość elektroniki jest produkowana w Chinach. Są niby smartfony wyprodukowane w Niemczech, ale jednak mają też chińskie części. Trochę podobnie jest z tymi archiwizatorami.

**Reply by hawkeye116477** [reply]
Skoro tak generalizujecie, to co w takim razie z `muve.pl` i grami wydanymi przez Cenegą, bojkotujecie?
Ich właściciel był rosyjski, a jakiś czas temu Chińczycy przejęli.
A może jednak nie trzeba być takim pesymistą i wrzucać wszystko do jednego wora. Przekazali przecież kasę na wsparcie Ukrainy, podobnie jak Wargaming, który wywodzi się z Białorusi.

**Reply by seba2282** [reply]
Ciekawe co mówisz. A masz źródła na to także?

**Reply by hawkeye116477** [reply]
O przejęciu jest np na `https://blizzplanet.pl/gaming/cenega-muve-pl-i-qloc-kupione-przez-tencent/`, a jeżeli chodzi o ich datki to jest już to w zaktualizowanym poście na dole razem z resztą interesujących lektur.

**Reply by seba2282** [reply]
Trzeba dodać domeny ru i su (soviet union) do KAD też 😂


Czytam też twoje lektury.

**Reply by hawkeye116477** [reply]
> do KAD

O nieco podobnej sprawie już toczy się dyskusja => https://github.com/FiltersHeroes/KAD/discussions/2070.

**Reply by AroKol78** [reply]
> Tym bardziej że sami Rosjanie nie zrobili wiele by nie było wojny

tak dla kontrastu i refleksji
```
https://www.polsatnews.pl/wiadomosc/2022-03-12/rosja-blokuje-instagram-placz-rosyjskiej-blogerki-to-cale-moje-zycie/
https://www.polsatnews.pl/wiadomosc/2022-03-11/wojna-rosja-ukraina-pacjentka-ostrzelanego-szpitala-polozniczego-w-mariupolu-urodzila-dziecko/
```

**Reply by seba2282** [reply]
> To jak tam odinstalowaliście już WinRara i 7-Zipa?
> 
> Tylko czym tu zastąpić, może programem PeaZip (twórca jest Włochem, ale to jest fork 7-Zipa) albo HaoZip (chiński)? Jeśli żaden z nich wam nie odpowiada, to lepszego wyboru chyba brak. No chyba, że jednak zgadzacie się z autorem artykułu `https://www.dobreprogramy.pl/@antar/rosyjskie-programy-i-dystrybucje-z-ktorych-nie-wypada-juz-korzystac,blog,115705` i nie wymieniacie ich.

Jest kilka ich tutaj i dlatego wystartowała cała dyskusja.

**Reply by hawkeye116477** [reply]
Znalazłem porównanie archiwizerów => `https://www.topattack.com/list/best-file-compression-software-review/35`, ale nie wiem na ile one wiarygodne, ja bym pewnie inaczej to ułożył.
Tak w zasadzie, to wychodzi na to, że z darmowych jest jeszcze Bandizip, ale ma on reklamy i ograniczenia, jZip instaluje jakieś opcjonalne śmieci, ale na pewno wykreślamy, bo dawno nie miał aktualizacji, ale za to jest jeszcze **IZArc**, który być może by się nadał. Oprócz tego mamy też B1 Free Archiver, ale ten to dopiero bardzo dawno nie miał aktualizacji, także wypada z gry.
A HaoZip jednak też odpada, bo długo nie aktualizowany.
Jest jeszcze **Explzh** nie ujęty w zestawieniu, ale jego strona internetowa jakoś mnie odpycha i do końca nie wiadomo co to za firma te pon software, prawdopodobnie Japończyki, a w ogóle też jakoś mało opinii o tym ich programie.

Podsumowując, jak dla mnie mamy 2 zwycięzców => **PeaZip** (włoski, otwartoźródłowy) i **IZArc** (bułgarski, zamkniętoźródłowy). No chyba, że wolicie za jakiś program zabulić, o ile jest taki jeden naprawdę wart tego.

**Reply by krystian3w** [reply]
tylko na *papierze. Bo raczej nie chodzi o umowy z Watykanem.

**Reply by hawkeye116477** [reply]
A może wypadałoby napisać pełnym zdaniem, by było wiadomo o co dokładnie biega :stuck_out_tongue:.

Tak czy siak, jak się zorientowałem najwyraźniej nie wszystko zostało stworzone przez autora PeaZipa, co jest obecne w programie. Do części formatów są wykorzystywane biblioteki, np do 7-Zipa. Ale tak samo jest w IZArcu i innych, bo Ruskie wymyślili m.in. formaty RAR i 7-zip. Tak więc w zasadzie każdy ma jakiś 1 własny format i swój interfejs graficzny, a do reszty wykorzystuje biblioteki stworzone przez innych.

**Reply by krystian3w** [reply]
Mnie chodziło o literówkę.

**Reply by AroKol78** [reply]
słusznie zauważony, mój błąd ortograficzny (`papierze` `papieże`)

**Reply by hawkeye116477** [reply]
Aha, ok.

**Reply by krystian3w** [reply]
Co do genezy zip to trzeba by poznać resztę co-invertorów poza tym co się urodził w Wisconsin (o drzewo genealogiczne, że ma krewnych z Alaski co byli Rosjanami może być trudno).

**Reply by hawkeye116477** [reply]
Ah, no tak, ZIPa wymyślił Amerykanin, a RARa (Roshal archive) Rosjanin, pomyliłem jedno z drugim, poprawiłem wpis.

**Comment by hawkeye116477** [comment]
Do poczytania dla zainteresowanych:
```
https://adguard.com/en/blog/official-response-to-setapp.html
https://www.reddit.com/r/Adguard/comments/t15gr4/announcement_on_the_topic_of_the_war_in_ukraine/
https://www.hotgear.pl/aplikacje/myslisz-o-bojkocie-rosyjskich-aplikacji-oto-lista-tych-najbardziej-znanych/
https://cdaction.pl/newsy/wojna-na-ukrainie-stanowisko-wargamingu-i-milion-dolarow-na-czerwony-krzyz
https://cenega.pl/news/grupa-qloc-cenega-oraz-muve-pl-przekazuja-1-milion-zlotych-na-wsparcie-humanitarne-ukrainy/8758
```

**Comment by hawkeye116477** [comment]
A co sądzicie o Nokii, bojkotujecie?

```
https://www.instalki.pl/aktualnosci/bezpieczenstwo/53226-nokia-rosja-podsluch-huawei-fsb-inwigilacja.html
https://www.chip.pl/2022/03/nokia-wycofuje-sie-z-rosji-zostawia-po-sobie-sprzet-do-inwigilacji-opozycji/
https://antyweb.pl/nokia-rosja-sorm
```

**Reply by seba2282** [reply]
Z tego co widzę Nokia podsłuchiwała obywateli Rosji. Więc trudno z bojkotować w dobie tego co teraz robią Rosjanie. 😂

**Reply by hawkeye116477** [reply]
A ja sądzę, że raczej pochwała im się nie należy... Z artykułu wynika, że było to robione na polecenie rządu rosyjskiego, zapewne by szpiegować osoby niewygodne politycznie, przeciwników.

**Comment by siepa3** [comment]
Boże, chroń mnie przed "przyzwoitymi" ludźmi.
